### DISCUSSION TITLE
Incomplete tests?

### DISCUSSION THREAD

**Discussion by Peter-developer01** [monkeytypegame/monkeytype#5978] [Questions] [answered]
What is the "incomplete tests" field in XP breakdown after a test? I don't get it.

**Comment by Peter-developer01** [comment]
Also, what does the "clean" value mean?

**Comment by fehmer** [comment, marked as answer]
"Incomplete tests" is XP you gained from test you did before but didn't finish for any reason, this is based on time.

"Clean" is bonus XP if you made errors but corrected them.

**Reply by Peter-developer01** [reply]
Thanks!
